wallet/createtx.go

```diff
@@ -629,11 +629,12 @@ func (w *Wallet) txToMultisigInternal(ctx context.Context, op errors.Op, dbtx wa
 	}
 	msgtx.AddTxOut(txOut)
 
-	// Add change if we need it. The case in which
-	// totalInput == amount+feeEst is skipped because
-	// we don't need to add a change output in this
-	// case.
-	feeSize := txsizes.EstimateSerializeSize(scriptSizes, msgtx.TxOut, 0)
+	// Add change if we need it.
+	changeSize := 0
+	if totalInput > amount+feeEstForTx {
```

Comment:
This isn't exactly right, since it could be the case that `totalInput` is greater than `amount+feeEstForTx` but lower than the dust limit, which shouldn't create a change amount, but since `feeEstForTx` is _significantly_ overestimated (0.5 dcr) this shouldn't happen. 

Fixing this edge case (dust limit change) would require tweaking this entire function to do a proper fee estimation, so unless this is really needed, the proposed change fixes the original issue.